Lib/json/tool.py

```diff
@@ -16,6 +16,18 @@
 import sys
 
 
+def parse_indent(indent):
+    """Parse the argparse indent argument."""
+    if indent == 'None':
+        return None
+    if indent == r'\t':
+        return '\t'
```

Comment:
I don't like such special casing.
I think `json.tool` should be minimal to expose `json.dumps` as CLI.